xbmc/addons/AddonBuilder.cpp

```diff
@@ -136,6 +138,8 @@ std::shared_ptr<IAddon> CAddonBuilder::Build()
       return CAudioEncoder::FromExtension(std::move(m_props), m_extPoint);
     case ADDON_AUDIODECODER:
       return CAudioDecoder::FromExtension(std::move(m_props), m_extPoint);
+    case ADDON_IMAGEDECODER:
+      return CImageDecoder::FromExtension(std::move(m_props), m_extPoint);
```

Comment:
There is no such method, at least there was none when i updated this last time. I had to *remove* FromProps in that update.